### DISCUSSION TITLE
Drean Textures 0.4.1 TypeError in Stable Diffusion XL: __init__() got an unexpected keyword argument 'controlnet'

### DISCUSSION THREAD

**Discussion by xvpowerg** [carson-katri/dream-textures#826] [Q&A]
https://github.com/user-attachments/assets/e2563a0d-402c-4bdd-91ea-bb9cba111921

 I encountered the following error when trying to use Stable Diffusion XL with ControlNet:
 TypeError("StableDiffusionXLPipeline.__init__() got an unexpected keyword argument 'controlnet'")

ControlNet model: controlnet-depth-sdxl-1.0
Running on Windows with an NVIDIA 4090

Has anyone else encountered this issue or found a solution for getting ControlNet to work with SDXL? Any help would be greatly appreciated!

**Comment by rukia5216** [comment]
sorry,but where did u put the controlnet model?
